wasatch/AndorDevice.py

```diff
@@ -484,7 +497,7 @@ def get_serial_number(self) -> SpectrometerResponse:
     def init_tec_setpoint(self) -> SpectrometerResponse:
         minTemp = c_int()
         maxTemp = c_int()
-        self.check_result(self.driver.GetTemperatureRange(byref(minTemp), byref(maxTemp)), "GetTemperatureRange")
+        self.check_result(self.driver.GetTemperatureRange(byref(minTemp), byref(maxTemp)), "GetTemperatureRange") # step 5
```

Comment:
what's with the step # comments?

Reply:
Those were added to align with similar comments I added here: 

https://github.com/WasatchPhotonics/Wasatch.NET/pull/108

In the future it'll be easier to compare the .NET -vs- Python implementations when tracing down divergent behavior between the two implementations.